packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -31,11 +31,24 @@ abstract class ProgressIndicator extends StatefulWidget {
   /// The [value] argument can be either null (corresponding to an indeterminate
   /// progress indicator) or non-null (corresponding to a determinate progress
   /// indicator). See [value] for details.
+  ///
+  /// {@template flutter.material.progressIndicator.semantics}
+  /// The [semanticsLabel] can be used to identify the purpose of this progress
+  /// bar for screen reading software. The [semanticsValue] property may be used
+  /// for determinate progress indicators to indicate how much progress has been made.
+  /// The [semanticsLiveRegion] parameter corresponds to [Semantics.liveRegion],
+  /// and should only be set to true if this progress indicator is a primary item
+  /// on the screen, as it will cause the screen reader to continuously and politely
+  /// read the status of this indicator.
+  /// {@endtemplate}
   const ProgressIndicator({
     Key key,
     this.value,
```

Comment:
Here and for the others: Why is the default value for `semanticsValue` not derived from the actual progress `value`?

Reply:
I was thinking people could use max values other than 1.0, which is not the case.  I'll default it.